RxJavaTemplate/app/src/test/java/co/nimblehq/rxjava/ui/screens/MainNavigatorTest.kt

```diff
@@ -33,7 +33,7 @@ class MainNavigatorTest {
     }
 
     @Test
-    fun `Should navigate to Second screen from Home correctly`() {
+    fun `When calling navigate responds successfully with destination as HomeFragment, it should navigate to Second Screen from Home Screen correctly`() {
```

Comment:
```suggestion
    fun `When navigating to Second screen from Home screen, it should navigate with corresponding bundle`() {
```
the condition and result are quite similar, I think we could omit it but still keep following the correct format